scripts/artfwo/glut.lua

```diff
@@ -359,5 +359,4 @@ function cleanup()
     poll.polls['phase_' .. v]:stop()
     poll.polls['level_' .. v]:stop()
```

Comment:
does the call to `poll.clear_all()` in `script.lua` make the poll stopping unneeded too?

https://github.com/monome/norns/blob/90b16dfa3a4d2114e448a4fb4558f1b8687680dd/lua/script.lua#L27


https://github.com/monome/norns/blob/90b16dfa3a4d2114e448a4fb4558f1b8687680dd/lua/script.lua#L27

Reply:
yeah, clear_all stops the polls too, so these lines can now be removed as well.

Reply:
cool.  looks like the whole `cleanup()` is unneeded then.  👍